packages/cosmic-swingset/src/kernel-stats.js

```diff
@@ -361,6 +381,46 @@ export function exportKernelStats({
     }
   }
 
+  let heapStatsLast = 0;
```

Comment:
I thought there'd be something easier.  Maybe document the fact that I tried...

```suggestion
  // TODO: We probably should not roll our own Node.js process metrics, but a cursory search for
  // "opentelemetry node.js VM instrumentation" didn't reveal anything really useful.
  let heapStatsLast = 0;
```